.docker/Dockerfile.alpine

```diff
@@ -0,0 +1,36 @@
+FROM node:12.22.8-alpine3.14
+
+RUN apk add --no-cache python3 make g++ libc6-compat ttf-dejavu
+
+ADD . /app
+
+MAINTAINER buildmaster@rocket.chat
+
+RUN set -x \
+ && cd /app/bundle/programs/server \
+ && npm install \
+ # Start hack for sharp...
+ && rm -rf npm/node_modules/sharp \
+ && npm install sharp@0.29.3 \
+ && mv node_modules/sharp npm/node_modules/sharp \
+ # End hack for sharp
+ && cd npm \
+ && npm rebuild bcrypt --build-from-source \
+ && npm cache clear --force
+
+# needs a mongo instance - defaults to container linking with alias 'mongo'
+ENV DEPLOY_METHOD=docker \
+    NODE_ENV=production \
+    MONGO_URL=mongodb://mongo:27017/rocketchat \
```

Comment:
@geekgonecrazy why's this being set here? 🤔

Reply:
imo it'd be better to not set this here. MONGO_URL, MONGO_OPLOG_URL are hard dependencies, 1. must be set, 2. can't be anything wrong

better to have the container spit out 'MONGO_(OPLOG_)?URL not found' in case it's not passed via compose or cli instead of falling back to something default.

Reply:
this is kinda a legacy from older Dockerfiles.. 

I think the idea was to have 3rd party tools to read those env vars were required and then provide a field on their UIs to provide the correct values.. and also give a quick start, since oplog was not required at that time..

but I agree with @debdutdeb points, it might be good to not have this set anymore.

Reply:
I’m not sure I see it being included as a bad thing.  But as long as we are certain it won’t break by removing.. I’m ok with removing